app/forms/support_interface/application_forms/edit_reference_details_form.rb

```diff
@@ -21,12 +21,14 @@ def self.build_from_reference(reference)
       def save(reference)
         return false unless valid?
 
-        reference.update!(
-          name:,
-          email_address:,
-          relationship:,
-          audit_comment:,
-        )
+        ApplicationForm.with_unsafe_application_choice_touches do
```

Comment:
Why do we think it's safe to use this here?

Reply:
Because if there are any changes on references it should be bumped over the API, even if is an old cycle